app/views/users/_repositories.html.erb

```diff
@@ -45,23 +49,29 @@
     <% end %>
 <% else %>
     <% repos.each do |joined_project| %>
-        <table class="spacing_table">
-            <thead>
-                <tr>
-                    <th>Project Proposal</th>
-                    <th>Action</th>
-                </tr>
-            </thead>
-            <tbody>
-                <tr>
-                    <td class="titlepp">
-                        <%= joined_project.project_proposal.title %>
-                    </td>
-                    <td>
-                        <%= link_to 'Show', joined_project.project_proposal, class: 'green-button' %>
-                    </td>
-                </tr>
-            </tbody>
-        </table>
+        <% project_proposal = joined_project.project_proposal %>
```

Comment:
Really not a fan of Ruby in HTML... For `has_user` you can create a method in the model